policy_agent/policy_agent.py

```diff
@@ -0,0 +1,643 @@
+#!/usr/bin/python
+import os
+import sys
+import time
+import json
+import Queue
+import logging
+import requests
+import pycalico
+
+from threading import Thread
+from subprocess import check_output
+from contextlib import closing
+from pycalico.datastore_datatypes import Rules, Rule
+from pycalico.datastore_errors import ProfileNotInEndpoint, ProfileAlreadyInEndpoint
+from pycalico.datastore import DatastoreClient, PROFILE_PATH
+
+KUBE_API_ROOT = os.environ.get('KUBE_API_ROOT',
+                               'http://localhost:8080/api/v1/')
+
+ANNOTATION_NAMESPACE = "projectcalico.org/"
```

Comment:
This is common between the plugin and the agent, and should be defined in an external file which is imported into each so that we don't end up with mismatches.

A new `constants.py` would do the trick.

Then, the plugin and the agent could do:
```
from constants import ANNOTATION_NAMESPACE
```